app/src/test/java/com/example/parkingsystemkotlin/mvp/presenter/ReservationParkingPresenterTest.kt

```diff
@@ -1,18 +1,34 @@
+@file:Suppress("DEPRECATION")
```

Comment:
please delete this, and will use the correct imports

Reply:
Done